packages/core/content-type-builder/admin/src/components/AIChat/lib/types/messages.ts

```diff
@@ -24,22 +23,25 @@ export interface TextContent {
 
 export type MessageContent = TextContent | MarkerContent;
 
-export type UserMessage = {
-  id: string;
-  revisionId?: string;
-  role: 'user';
-  contents: TextContent[];
-  attachments: Attachment[];
-  project?: string;
-};
-
-export type AssistantMessage = {
-  id: string;
-  revisionId?: string;
-  role: 'assistant';
-  schemaChanges: SchemaChange[];
-  contents: MessageContent[];
-  status: Status;
-};
-
-export type Message = UserMessage | AssistantMessage;
+// export type UserMessage = {
```

Comment:
since we are using UIMessage typings now we probably can remove them instead of keeping those commented